README.md

```diff
@@ -408,9 +406,9 @@ This is `false` by default because it should only be *intentionally* turned on,
 
 ### htmlSource
 
-Type: `string`. Absolute path, please. Default: `${project-root}/src/html.js`.
+Type: `Function`.  Default: `[Default HTML](https://github.com/mapbox/underreact/blob/next/lib/default-html.js)`.
 
-The path to your HTML source file. For more information, read [`Defining your HTML`](#defining-your-html).
+The function to generate your apps HTML template. For more information, read [`Defining your HTML`](#defining-your-html).
```

Comment:
Let's be explicit here that the recommended approach is to create a separate module and `require` it.